Makefile

```diff
@@ -67,11 +68,19 @@ $(SRCDIR64)/%.elf: $(SRCDIR64)/%.$(SEXT)
 
 $(SRCDIR32)/%.elf: $(SRCDIR32)/%.$(SEXT) 
 	riscv64-unknown-elf-gcc -g -o $@ -march=rv32gq$(CMPR_FLAGS)_zfa_zba_zbb_zbc_zbs_zfh_zicboz_zicbop_zicbom_zicond -mabi=ilp32 -mcmodel=medany \
-	    -nostartfiles -T${WALLY}/examples/link/link.ld $<
+	    -nostartfiles -T${WALLY}/examples/link/link.ld -I${WALLY}/tests/coverage $<
```

Comment:
This should either only be added to privilege tests or to both rv32 and rv64, not just privilege and rv32